src/coreclr/pal/src/arch/amd64/context2.S

```diff
@@ -183,6 +183,26 @@ LOCAL_LABEL(Done_Restore_CONTEXT_FLOATING_POINT):
     kmovq k6, qword ptr [rdi + (CONTEXT_KMask0 + 6 * 8)]
     kmovq k7, qword ptr [rdi + (CONTEXT_KMask0 + 7 * 8)]
 
+    test    BYTE PTR [rdi + CONTEXT_XStateFeaturesMask], XSTATE_MASK_APX
+    je      LOCAL_LABEL(Done_Restore_CONTEXT_XSTATE)
+
+    mov r16, qword ptr [rdi + CONTEXT_Egpr + 0 * 8]
+    mov r17, qword ptr [rdi + CONTEXT_Egpr + 1 * 8]
+    mov r18, qword ptr [rdi + CONTEXT_Egpr + 2 * 8]
+    mov r19, qword ptr [rdi + CONTEXT_Egpr + 3 * 8]
+    mov r20, qword ptr [rdi + CONTEXT_Egpr + 4 * 8]
+    mov r21, qword ptr [rdi + CONTEXT_Egpr + 5 * 8]
+    mov r22, qword ptr [rdi + CONTEXT_Egpr + 6 * 8]
+    mov r23, qword ptr [rdi + CONTEXT_Egpr + 7 * 8]
+    mov r24, qword ptr [rdi + CONTEXT_Egpr + 8 * 8]
+    mov r25, qword ptr [rdi + CONTEXT_Egpr + 9 * 8]
+    mov r26, qword ptr [rdi + CONTEXT_Egpr + 10 * 8]
+    mov r27, qword ptr [rdi + CONTEXT_Egpr + 11 * 8]
+    mov r28, qword ptr [rdi + CONTEXT_Egpr + 12 * 8]
+    mov r29, qword ptr [rdi + CONTEXT_Egpr + 13 * 8]
+    mov r30, qword ptr [rdi + CONTEXT_Egpr + 14 * 8]
+    mov r31, qword ptr [rdi + CONTEXT_Egpr + 15 * 8]
```

Comment:
We shouldn't strictly need `LLVM 19` here (or `LLVM 20` for AVX10.2).

Both MASM and GAS have directives that allow defining macros and directives that allow emitting raw bytes.

Correspondingly, we should be able to either just do `.byte 0x00, 0x00, 0x00` (using appropriate bytes of course) and add a comment indicating what it translates to

Reply:
Thanks for the inputs!

So we can replace these asm code with raw hex code for now to get the CI path, am I understanding it right?

Reply:
Thanks!